temp.js

```diff
@@ -0,0 +1,138 @@
+const {
+    Worker, isMainThread, workerData, BroadcastChannel, MessageChannel,
+  } = require('worker_threads');
+  const CoinbasePro = require('coinbase-pro');
+  const stringTable = require('string-table');
+  const moment = require('moment');
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 0 spaces but found 2.](https://app.codacy.com/gh/amcalabretta/botbase/pullRequest?prid=7628479)
```suggestion
const moment = require('moment');
```